fusequery/query/src/api/rpc/flight_scatter.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020-2021 The Datafuse Authors.
+//
+// SPDX-License-Identifier: Apache-2.0.
+
+use std::sync::Arc;
+
+use common_datablocks::DataBlock;
+use common_datavalues::DataField;
+use common_datavalues::DataSchemaRef;
+use common_datavalues::DataSchemaRefExt;
+use common_datavalues::DataType;
+use common_datavalues::DataValue;
+use common_exception::ErrorCodes;
+use common_exception::Result;
+use common_planners::Expression;
+
+use crate::pipelines::transforms::ExpressionExecutor;
+
+pub struct FlightScatter(Arc<ExpressionExecutor>, String, usize);
+
+impl FlightScatter {
+    pub fn try_create(
+        schema: DataSchemaRef,
+        action: Expression,
+        num: usize
+    ) -> Result<FlightScatter> {
+        let indices_expression_action = Expression::ScalarFunction {
+            op: String::from("modulo"),
```

Comment:
Does it only scatted by hash?
If there are some others scatter in future, it would better to rename `FlightScatter` to `FlightScatterByHash`?

Reply:
Done.